src/formElementPlugins/imageDisplay/src/SummaryView.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react'
 import { Image } from 'semantic-ui-react'
+import { allowedSizeValues } from './ApplicationView'
 import { SummaryViewProps } from '../../types'
 
 const SummaryView: React.FC<SummaryViewProps> = ({ parameters }) => {
-  const { url, size, alignment, altText } = parameters
+  const { url, size, alignment, altText, summarySize = 'small', style = {} } = parameters
+
+  // By default, image is displayed "small" in Summary view, but can be
+  // over-ridden by specifying "summarySize"
```

Comment:
There are cases where we need Summary view to not be restricted to "small" (such as in Data views), so need to allow this to be overridden

(This oversight actually explains a lot of the issues I've had trying to get the Image plugin to display correctly sometimes, so glad to have fixed it)